addons/core/addon/components/form/authenticate/index.hbs

```diff
@@ -3,12 +3,10 @@
   SPDX-License-Identifier: MPL-2.0
 }}
 
-{{! //TODO fix the issue and delete disabling the rule }}
-{{! template-lint-disable no-passed-in-event-handlers }}
 {{#if @model.type}}
   {{component
     (concat 'form/authenticate/' @model.type)
-    submit=@submit
+    onSubmit=@submit
```

Comment:
None blocking comment: We use `onSubmit` without the `@` i.e: `@onSubmit` because is a component declaration?

Reply:
@calcaide yes that is correct